core/src/com/ericc/the/game/map/Room.java

```diff
@@ -39,4 +47,9 @@ public boolean equals(Object o) {
 
         return false;
     }
+
+    public int getMinimalSize() {
```

Comment:
nie jest jasne z nazwy co to zwraca. Chyba chodzi o dimension a nie size? I chyba nie minimal tylko, dunno, shorter? MinimalSize brzmi jak najmniejszy rozmiar jaki może mieć mapa (czyli np. 5x5)